### PR TITLE
Add Rooms Page

### PR DESCRIPTION
**Pull request by hpneo**
Renamed previous `<RoomsPage />` to `<RoomDetailPage />` and add rooms list page.

### CODE REVIEW

File: src/components/RoomsPage/RoomsPage.js [hunk 1]
```diff
@@ -1,25 +1,107 @@
...
-import Tabs, { TabItem } from "components/shared/Tabs";
+import CircularButton from "components/shared/CircularButton";
+import DropdownMenu from "components/shared/DropdownMenu";
+import { ReactComponent as ListIcon } from "icons/list.svg";
+import { ReactComponent as GridIcon } from "icons/grid.svg";
+import { ReactComponent as GlobeIcon } from "icons/globe-public.svg";
+import { ReactComponent as MoreIcon } from "icons/more.svg";
 import styles from "./RoomsPage.module.css";
 
+const fakeRooms = [
```
Comment (alessandrapereyra) [inline src/components/RoomsPage/RoomsPage.js:19]: This is one of the items we'd be moving to the mock API, right?

File: src/components/RoomsPage/RoomsPage.js [hunk 2]
```diff
@@ -29,52 +111,56 @@ const RoomsPage = () => {
...
-          <Route exact path={`${path}/content`} component={ContentPage} />
-          <Route exact path={`${path}/members`} component={MembersPage} />
-          <Route
-            exact
-            path={`${path}/live-insights`}
-            component={LiveInsightsPage}
-          />
-          <Route exact path={`${path}/moderation`} component={ModerationPage} />
-        </Switch>
+        <Grid cols={4} gap={24}>
```
Comment (alessandrapereyra) [inline src/components/RoomsPage/RoomsPage.js:132]: Wondering whether we should instead use 

```suggestion
        <Grid cols="4" gap="24">
```

And perhaps move them towards constants.

File: src/components/RoomsPage/RoomsPage.js [hunk 3]
```diff
@@ -29,52 +111,56 @@ const RoomsPage = () => {
...
+            <Card
+              key={room.id}
+              cover={
+                <Link to={`${url}/${room.id}`}>
+                  <figure style={{ margin: 0, position: "relative" }}>
+                    <CircularButton className={styles.cardIcon}>
+                      <GlobeIcon width="20" height="20" />
+                    </CircularButton>
+                    <img
+                      src={`https://picsum.photos/seed/${room.id}/260/190`}
```
Comment (alessandrapereyra) [inline src/components/RoomsPage/RoomsPage.js:143]: Smart.

File: src/components/RoomsPage/RoomsPage.js [hunk 4]
```diff
@@ -29,52 +111,56 @@ const RoomsPage = () => {
...
+                    />
+                  </figure>
+                </Link>
+              }
+            >
+              <Stack distribution="between">
+                <Heading as="h5">
+                  <Link to={`${url}/${room.id}`}>{room.fullName}'s Room</Link>
```
Comment (alessandrapereyra) [inline src/components/RoomsPage/RoomsPage.js:151]: Good one. I'm not sure if " 's Room" is something we should be following vs assuming that substring will be part of the name. Perhaps Will can chime in here.

File: src/components/shared/DropdownMenu/DropdownMenu.js [hunk 5]
```diff
@@ -42,7 +42,7 @@ const DropdownMenu = ({
           >
             {children}
           </IconButton>
-          {isOpen && (
+          {isOpen && items.length > 0 && (
```
Comment (alessandrapereyra) [inline src/components/shared/DropdownMenu/DropdownMenu.js:45]: Would this be preferred vs `items.length`? 

I've been using the latter but perhaps I shouldn't?

File: src/components/RoomsPage/RoomsPage.js (hunk 1 above)
Comment (hpneo) [reply src/components/RoomsPage/RoomsPage.js:19]: Right.

File: src/components/RoomsPage/RoomsPage.js (hunk 4 above)
Comment (hpneo) [reply src/components/RoomsPage/RoomsPage.js:151]: That's something we can work on when we move the fake data to the endpoint. I'm being lazy here.

File: src/components/shared/DropdownMenu/DropdownMenu.js (hunk 5 above)
Comment (hpneo) [reply src/components/shared/DropdownMenu/DropdownMenu.js:45]: Sure. It's the same.

**Pull request merged**